benchmarks/go.mod

```diff
@@ -15,6 +16,9 @@ require (
 	gopkg.in/inconshreveable/log15.v2 v2.0.0-20221122034931-555555054819
 )
 
+//	fortio.org/log => ../../../fortio.org/log
+replace go.ubder.org/zap => ../
```

Comment:
This entry already exists at the top of the file.

Reply:
opsa! I was looking at the go.uber.org/zap v1.23.0 in require and wanted to check I was running the latest, but didn't see the existing replace, thx